README.md

```diff
@@ -1 +1,54 @@
-# Awesome-books-with-ES6
\ No newline at end of file
+# Awesome Books With ES6
+
+> Creating an app that adds and delete books from a list.
+
+<p align="center">
+  <img src="./screenshot.png">
+</p>
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Application Page url
+
+- [Live Demo Link](https://cynthiainga.github.io/Awesome-books-with-ES6/)
+
+## Getting Started
+
+> To get a local copy follow the following steps:
+
+- Copy this link https://github.com/cynthiainga/Awesome-books-with-ES6.git
+- Open your terminal or command line
+- Run git clone and paste the link
+- Open the folder with your code editor
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.

Reply:
Already changed